docs/faststore/docs/customization/building-sections/override-use-cases/adding-more-icon-options-to-the-alert-component.mdx

```diff
@@ -0,0 +1,125 @@
+
+Add the section to the Headless CMS by following the instructions available in [syncing components to the Headless CMS](https://developers.vtex.com/docs/guides/faststore/overrides-syncing-components-with-the-headless-cms). The following schema was used as an example:
+
+        ```json
+        [    
+            {
+            "name": "CustomIconsAlert",
+            "schema": {
+                "title": "Alert with more Icon options",
+                "description": "Add an alert that has more Icon options",
+                "type": "object",
+                "required": ["icon", "content", "dismissible"],
+                "properties": {
+                "icon": {
+                    "type": "string",
+                    "title": "Icon",
+                    "enumNames": [
+                    "Bell",
+                    "BellRinging",
+                    "Checked",
+                    "Info",
+                    "Truck",
+                    "User",
+                    "Plus",
+                    "PlusCircle",
+                    "Ruler",
+                    "Star",
+                    "Storefront",
+                    "Twitter"
+                    ],
+                    "enum": [
+                    "Bell",
+                    "BellRinging",
+                    "Checked",
+                    "Info",
+                    "Truck",
+                    "User",
+                    "Plus",
+                    "PlusCircle",
+                    "Ruler",
+                    "Star",
+                    "Storefront",
+                    "Twitter"
+                    ]
+                  }
+                }
+              }
+            }
+        ]
+        ```
+
+> ℹ️ For further details on code implementation, see the [`sections.json`](https://github.com/vtex-sites/playground.store/blob/main/cms/faststore/sections.json#L1-L55) file available in the [playground.store](https://github.com/vtex-sites/playground.store) repository.
+
+## Results
+
+Once you have [set your development mode](https://developers.vtex.com/docs/guides/faststore/headless-cms-previewing-changes-in-development-mode) to see the changes locally, you can see the new Alert icon. Below, you can check an example storefront before and after the instructions in this guide:
+
+- **Before**
+
+    The native Alert displays one of the default icons, `BellRinging`, alongside the text `Get 15% off today: NEW15! Buy now`.
+
+    ![before-custom-icon](https://vtexhelp.vtexassets.com/assets/docs/src/before-alert-icon___86e395e9cfc731f1f70b56cf3760af08.png)
+
+- **After**
+  
+    The native Alert was replaced with the custom one, which brings a new icon, `Star`, from the `@faststore/ui` [iconography](https://developers.vtex.com/docs/guides/faststore/reference-icons#usage) library, alongside the text `Rate your favorite item`.
+
+    ![after-custom-icon](https://vtexhelp.vtexassets.com/assets/docs/src/after-custom-alert-icon___87cd248c0c406640aaaac4ef9238e101.png)
```

Comment:
in my opinion, the banner catches too much attention. it took me a while to see the alert on top and see the icon. i would suggest cropping the image to get only the header and the alert.

Reply:
Makes sense, and I adjusted the image.